doc/nrf/known_issues.rst

```diff
@@ -659,6 +659,14 @@ Closing sockets
 Multiprotocol Service Layer (MPSL)
 ==================================
 
+.. rst-class:: v1-4-0 v1-3-2 v1-3-1 v1-3-0
```

Comment:
@greg-fer Shall we add the `v1-4-99-dev1` version, or will this happen at a later phase?

I will force push to check the newly added CI-doc-build label.

Reply:
It should have v1-4-99-dev1 if it applies to the dev tag.